semanticcpg/src/main/scala/io/shiftleft/semanticcpg/codedumper/CodeDumper.scala

```diff
@@ -49,9 +43,12 @@ object CodeDumper {
         method
           .collect {
             case m: Method if m.lineNumber.isDefined && m.lineNumberEnd.isDefined =>
-              val rawCode = if (lang == Languages.GHIDRA) {
-                m.code
-              } else {
+              val rawCode = if (lang == Languages.GHIDRA) { m.code }
```

Comment:
that indentation seems a little off and doesn't help readability - I guess scalafmt will complain as well...?

Reply:
Its fine for scalafmt.